build-system/pr-check.js

```diff
@@ -225,6 +225,11 @@ const command = {
     // Disabled because it regressed. Better to run the other saucelabs tests.
     // execOrDie(`${gulp} test --nobuild --saucelabs --oldchrome --compiled`);
   },
+  runVisualDiffTests: function() {
+    // This must only be run for push builds, since Travis hides the encrypted
+    // environment variables required by Percy during pull request builds.
+    execOrDie(`${gulp} visual-diff`);
```

Comment:
Looks like this will cause the build to fail if `visual-diff` returns status 1, which can happen if Percy is misconfigured or a runtime error is thrown. We probably want a "or-don't-die" variant for now.